<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="menuLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="captureButton">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="screenFrame">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="screenLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="morpologyEnableCheck">
        <property name="text">
         <string>Morpology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="horizontalFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioBallButton">
           <property name="text">
            <string>Ball</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioRobotButton">
           <property name="text">
            <string>Robot</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetColorButton">
        <property name="text">
         <string> Reset Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setBoardAreaButton">
        <property name="text">
         <string>Set Board Area
(자유형)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setBoardRectangleButton">
        <property name="text">
         <string>Set Board Area
(사각형)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>로봇 위치</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="robotSideSlider">
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bluetoothComboBox"/>
      </item>
      <item>
       <widget class="QFrame" name="horizontalFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="serialEditText"/>
         </item>
         <item>
          <widget class="QPushButton" name="serialSendButton">
           <property name="text">
            <string>send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="verticalFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Ball Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ballPosLabel">
           <property name="text">
            <string>Not found</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
